Classes/Point.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Classes
+{
+    //3
+    internal class Point
+    {
+        private int x, y;
+        private string namePoint;
+
+        public int xPoint
```

Comment:
Name it `X` (all properties and methods name should start with a capital letter)